libs/providers/launchdarkly-client/src/lib/launchdarkly-client-provider.ts

```diff
@@ -29,78 +32,114 @@ function wrongTypeResult<T>(value: T): ResolutionDetails<T> {
   };
 }
 
+/*
+ * Provider Status
+ * defined here https://github.com/open-feature/js-sdk/blob/d26c4eeedde9cc321c69375594f1ccbfaff41a28/packages/shared/src/types.ts#L265
+ * ideally will be exported by '@openfeature/web-sdk'
+ * */
+enum ProviderStatus {
+  NOT_READY = 'NOT_READY',
+  READY = 'READY',
+  ERROR = 'ERROR',
+}
+
```

Comment:
This is exported in the last few versions of the web-sdk. You can feel free to update to the latest web-sdk in the root package json of this project.

Reply:
I updated the package version to `@openfeature/web-sdk`, and I still get a compilation error. 
```Error during bundle: Error: libs/providers/launchdarkly-client/src/lib/launchdarkly-client-provider.ts:13:3 - error TS2305: Module '"@openfeature/web-sdk"' has no exported member 'ProviderStatus'.

13   ProviderStatus,```

wondering if I am missing something, I see it in the `common` package.

Reply:
Perhaps try deleting your node modules, and trying again? Locally, when I install this, I can import `ProviderStatus` without issue. When I `go to definition` in my IDE, I see:

![image](https://github.com/open-feature/js-sdk-contrib/assets/25272906/5928b79a-38de-432a-8ea7-056122a10bcc)

Which seems right to me.

Reply:
You also might have to clear any node modules in the library folder. I generally don't keep these around and try to avoid installing them at all.

Reply:
Indeed, thank you !!